.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,8 @@
+angular-boilerplate/styles.ef46db3751d8e999.css,1648459042712,46b50c321b39e89a491b6727a01628c34245605a30beb3e7414c5e01cff90e6e
```

Comment:
@Mubashir-Ansari stop tracking any cache files and move them to gitignore:
https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign